pkg/modules/authorization_deployment_scaffold_test.go

```diff
@@ -24,12 +24,13 @@ func TestGetProxyServerScaffold(t *testing.T) {
 	proxyImage := "proxy-image:latest"
 	opaImage := "opa-image:latest"
 	opaKubeMgmtImage := "kube-mgmt-image:latest"
+	configSecretProviderClassName := "array-creds"
```

Comment:
Tests can be updated to use configSecretName.

Reply:
Fixed